scripts/ci.js

```diff
@@ -22,6 +25,74 @@ const execaOpts = {
   }
 };
 
+const isHeadCommitOfPR = async () => {
+  const prNumber = process.env.CODEBUILD_SOURCE_VERSION.replace('pr/', '');
+
+  const { stdout } = await execa('curl', [
+    '-H',
+    '"Authorization:',
+    'token',
+    `${process.env.GITHUB_TOKEN}"`,
+    `https://api.github.com/repos/FormidableLabs/badges/pulls/${prNumber}/commits`
+  ]);
+
+  const headCommit = JSON.parse(stdout).pop().sha;
```

Comment:
Should we add a `try/catch` here for a more useful error message about what we're attempting in case we get a failure / odd `stdout` response?

Reply:
I believe this is resolved by going to using the octokit sdk.

Reply:
I believe this is resolved by using the octokit sdk.